docs/en_US/Compression/QuantizationSpeedup.rst

```diff
@@ -0,0 +1,113 @@
+Speed up Mixed Precision Quantization Model (experimental)
+==========================================================
+
+
+Introduction
+------------
+
+Deep learning network has been computational intensive and memory intensive 
+which increase the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like haq(https://arxiv.org/pdf/1811.08886.pdf) only support simulated mixed precision quantization which will 
+not speed up the inference process. To get real speedup of mixed precision quantization and 
+help people get the real feedback from hardware, we design a general framework with simple interface to allow NNI to connect different 
+DL model optimization backends (e.g., TensorRT, NNFusion), which gives users an end-to-end experience that after quantizing their model 
+with quantization algorithms, the quantized model can be directly speeded up with the connected optimization backend. NNI connects 
+TensorRT at this stage, and will support more backends in the future.
+
+
+Design and Implementation
+-------------------------
+
+To support speeding up mixed precision quantization, we divide framework into two part, frontend and backend.  
+Frontend could be popular training frameworks such as PyTorch, TensorFlow etc. Backend could be inference 
+framework for different hardware, such as TensorRT At present, we support PyTorch as frontend and 
+TensorRT as backend. To convert PyTorch model to TensorRT engine, we leverage onnx as intermediate graph 
+representation. In this way, we convert PyTorch model to onnx model, then TensorRT parse onnx 
+model to generate inference engine. 
+
+
+For the purpose of getting mixed precision engine, we need to provide a PyTorch model, calibrate dataset 
+and corresponded bit config. The calibrate set is used to calibrate quantized module. What' s more, the 
+bit config a dictionary whose key is layer name and value is bit width. After getting mixed precision engine, 
+we can do inference with given input data.
+
+Prerequisite
+------------
+CUDA version >= 11.0
+
+TensorRT version >= 7.2
+
+Usage
+-----
+
+.. code-block:: python
+
+    # arrange bit config for quantized layer
+    config = {
+        'conv1':8,
+        'conv2':32,
+        'fc1':16,
+        'fc2':8,
+    }
```

Comment:
suggest to show a relatively complete pipeline. give a model -> run quantization algorithm -> output/save the result model -> feed the model to modelspeedup -> get optimized model